packages/odyssey-react-mui/src/@types/react-augment.d.ts

```diff
@@ -11,13 +11,17 @@
  */
 
 import { FC } from "react";
-
 export interface ForwardRefWithType extends FC<WithForwardRefProps<Option>> {
   <T extends Option>(props: WithForwardRefProps<T>): ReturnType<
     FC<WithForwardRefProps<T>>
   >;
 }
 
-export type FocusHandle = {
-  focus: () => void;
-};
+type DataAttributeKey = `data-${string}`;
+declare module "react" {
+  interface InputHTMLAttributes<T> extends HTMLAttributes<T> {
```

Comment:
I will approve, but please check if the `type` definition at line 20 is supposed to go inside the `declare` at line 21.

I didn't notice it before, but I'm thinking that's how you localize global type overrides.